react-ui/public/index.html

```diff
@@ -24,6 +24,9 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css"
+      integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous"
+/>
```

Comment:
nit: format

Reply:
You can remove this line as it already present on US12 branch.

https://github.com/ser516asu/SER516-Team-Cleveland/blob/cc1c8b90f4d85a11b4c72f6476e3673eb9b25aa7/react-ui/public/index.html#L17